smurfs/src/reducers/index.js

```diff
@@ -21,3 +30,40 @@
   There is no need for 'combineReducers' in this project.
   Components can then read your store as, `state` and not `state.fooReducer`.
 */
+
+export const smurfsReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case FETCH_SMURFS:
+      return {
+        ...state,
+        err: "",
+        fetchingSmurfs: true
+      };
+    case CREATE_SMURF:
```

Comment:
I would also match verb type and tense here...If you are going to CREATE_SMURF, make a CREATE_SMURF_SUCCESS and a CREATE_SMURF_FAILURE as well...

Reply:
but what you have works...I'm just talking about things I have seen in the wild. :) More readable code...